lib/ui/painting.dart

```diff
@@ -2256,16 +2249,15 @@ class PathMetricIterator implements Iterator<PathMetric> {
 /// [PathMetric] objects.
 ///
 /// Once created, the methods on this class will only be valid while the
-/// iterator is at the contour for which they were created. When the next
-/// contour's [PathMetric] is obtained, the [length] and [isClosed] properties
-/// remain valid, but the [getTangentForOffset] and [extractPath] will throw a
-/// [StateError].
-// TODO(dnfield): Fix this if/when https://bugs.chromium.org/p/skia/issues/detail?id=8721 lands.
+/// iterator is at the contour for which they were created. It will also only be
+/// valid for the path as it was specifed when [Path.computeMetrics] was called.
+/// If additional contours are added or any contours are updated, the metrics
+/// need to be recomputed.
```

Comment:
what will happen if they're not?

Reply:
I think what's really going on here is that the API takes a snapshot of the path when you call it. In which case, we should just say that ("[Path.computeMetrics] measures the [Path] as it stands when the method is called; subsequently changing the [Path] does not affect the objects returned from the method." or something).

Reply:
Yes - I can update that to clarify.